src/core/text/Text.js

```diff
@@ -144,8 +144,15 @@ export default class Text extends Sprite
         const maxLineWidth = measured.maxLineWidth;
         const fontProperties = measured.fontProperties;
 
-        this.canvas.width = Math.ceil((width + (style.padding * 2)) * this.resolution);
-        this.canvas.height = Math.ceil((height + (style.padding * 2)) * this.resolution);
+        const prevTextWidth = this.textWidth;
+        const prevTextHeight = this.textHeight;
+
+        this.textWidth = Math.ceil(width + (style.padding * 2));
```

Comment:
there's no need to create 2 new member variables here, when you could just check against this.canvas.width and this.canvas.height

Reply:
I create 2 new member variables , because they could reduce  two `/ this.resolution` operations in `_generateFillStyle`.